torch/utils/data/distributed.py

```diff
@@ -47,7 +47,7 @@ class DistributedSampler(Sampler):
         ...     train(loader)
     """
 
-    def __init__(self, dataset, num_replicas=None, rank=None, shuffle=True, seed=0):
+    def __init__(self, dataset, num_replicas=None, rank=None, shuffle=True, seed=1):
```

Comment:
Why does changing the random seed from 0 to 1 change any behavior here? `g.manual_seed(0)` is still valid for reproducibility purposes, no?